react/src/components/AgentList.tsx

```diff
@@ -640,7 +645,7 @@ const AgentList: React.FC<AgentListProps> = ({
               values={[
                 { label: 'Agent' },
                 {
-                  label: record?.version,
+                  label: record?.version as string,
```

Comment:
Below is more safe code.
```suggestion
                  label: record?.version || '',
```